### DISCUSSION TITLE
AdvancedImageBox Control (Zoom, Pan, Regions, Selection, Cursor image)

### DISCUSSION THREAD

**Discussion by sn4k3** [AvaloniaUI/Avalonia#6080] [General]
Some time ago I've ported a amazing ImageBox control to Avalonia, being using it on my project since switched from .NET Forms to Avalonia and required a fast ImageBox that support advanced features, such as high zoom and pan with good performance. For a while i polished the control and kept only on the main project, but i think is a waste to keep it only for myself/my project. So i created a library with it and publish as nuget package, so others can also take advantage of this control. 
For docs you can either see original control documentation as i keep the most functions and names. Or explore UVtools code where i heavy use that control. Or just use Visual Studio code completion, functions are self explanatory. Hope someone can find this usefull. 

https://github.com/sn4k3/UVtools/tree/master/UVtools.AvaloniaControls
https://www.nuget.org/packages/UVtools.AvaloniaControls

* **AdvancedImageBox**
   * Port of: https://github.com/cyotek/Cyotek.Windows.Forms.ImageBox
   * Demo: https://youtu.be/bIr6P4dDlHc
   * Features:
      * Image modes (Zoom, fit, strech)
      * Pan & zoom
      * Select regions
      * Pixel grid
      * Cursor images
      * Configurable
      * Fast
  
![image](https://user-images.githubusercontent.com/113281/122132458-cb558c00-ce32-11eb-93e1-13834647792d.png)

**Comment by kekekeks** [comment]
Looks awesome, make a PR to https://github.com/AvaloniaCommunity/awesome-avalonia list

**Reply by sn4k3** [reply]
Have done, thank you :)
